modules/vscode-extensions/vscode-lsp/client/src/lspModules/oAuthService.ts

```diff
@@ -0,0 +1,87 @@
+import * as express from "express";
+import { Server } from "http";
+import { URL } from "url";
+import * as vscode from 'vscode';
+export class Wso2OAuth {
+	public app: express.Express;
+	public server: Server;
+
+	constructor(public port: number) {
+		this.app = express();
+		this.app.use(express.json(), express.urlencoded({ extended: false }));
+	}
+
+	public async StartProcess() {
+		
+		const host = new URL("http://localhost:8010");
+
+		this.server = this.app.listen(this.port);
+		this.app.get("/oauth", async (req, res) => {
+			const axios = require('axios');
+			try {
+				const clientID = '0aT4RyRSYOqsBKwsbmHsIcYwq1ca';
+				const clientSecret = 'VffkV69lSXw1FysVJYojxshSi9Ea';
+				let data = clientID + ':' + clientSecret;
+				let buff = new Buffer(data);
+				let base64data = buff.toString('base64');
+				const requestToken = req.query.code;
+				console.log(requestToken);
+				process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = "0";
```

Comment:
We call "0" as a magic number. We need to define a meaningful constant on the beginning of this file